lib/comgr/CMakeLists.txt

```diff
@@ -295,12 +295,14 @@ if (NOT UNIX)
     PRIVATE version)
 endif()
 
-enable_testing()
```

Comment:
Is removing this relying on the behavior described at https://cmake.org/cmake/help/v3.19/command/enable_testing.html:

> This command is automatically invoked when the CTest module is included, except if the BUILD_TESTING option is turned off.

?

We currently only require CMake 3.13.4, and don't want to bump it any further until upstream LLVM does, which doesn't document that behavior (see https://cmake.org/cmake/help/v3.13/command/enable_testing.html; it seems to have been added in 3.15?)

If it was just undocumented in 3.13.4 maybe we can cheat a little, but I'd prefer we just add the call back in for now, and maybe note in a `FIXME` that we can delete it when LLVM moves?

Reply:
Seems like it has been there for 15 years at least since version v2.4.0: https://github.com/Kitware/CMake/commit/e26c160698482b5c1c51746a62a185790f2af4a9

Reply:
Thanks for digging this up, I'm happy to rely on something that has been true since `CTest` has existed.